Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -518,9 +518,16 @@ extension AVFoundationPlayback {
     }
 
     open override var usingDVR: Bool {
+        guard playbackType == .live else { return false }
         guard let currentTime = player?.currentTime().seconds else { return false }
-        guard let seekableEndTime = seekableTimeRanges.first?.timeRangeValue.end.seconds else { return false }
-        return playbackType == .live && currentTime < seekableEndTime
+
+        for range in seekableTimeRanges {
```

Comment:
I think we can inline this _for_ using something like this:

`return seekableTimeRanges.first(where: { $0.timeRangeValue.end.seconds > currentTime})  != nil`